src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        System.out.println("Ball 1: " + ball1);
+        Ball ball2 = Lottery.getRandomBall();
+        System.out.println("Ball 2: " + ball2);
+        Ball ball3 = Lottery.getRandomBall();
+        System.out.println("Ball 3: " + ball3);
```

Comment:
create one lottery instance
call it in a loop
using numbers in variables' names is ofter a bad practice
make sure to read the common mistakes checklist before submitting ur solution :)